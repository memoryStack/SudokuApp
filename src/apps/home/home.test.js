import React from 'react';
import renderer from 'react-test-renderer'
import { Home } from '.'

jest.useFakeTimers()
test('renders correctly', () => {
  const homeTree = renderer.create(<Home />).toJSON()
  expect(homeTree).toMatchSnapshot()
})
