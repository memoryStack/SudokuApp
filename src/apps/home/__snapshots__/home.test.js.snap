// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  onLayout={[Function]}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "white",
      "height": "100%",
      "width": "100%",
    }
  }
>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#4088da",
        "borderRadius": 3,
        "justifyContent": "center",
        "opacity": 1,
        "paddingHorizontal": 8,
        "paddingVertical": 8,
        "top": 200,
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "white",
            "fontFamily": "ProximaNova-Regular",
            "fontSize": 20,
          },
          null,
        ]
      }
    >
      Start Game
    </Text>
  </View>
</View>
`;
